rest/admin_api_test.go

```diff
@@ -1178,32 +1189,175 @@ func TestDBOfflineSingleResync(t *testing.T) {
 	require.NoError(t, base.JSONUnmarshal(response.Body.Bytes(), &body))
 	goassert.True(t, body["state"].(string) == "Offline")
 
-	input := bytes.NewBufferString("")
-	request, _ := http.NewRequest("POST", "http://localhost/db/_resync", input)
+	response = rt.SendAdminRequest("POST", "/db/_resync?action=start", "")
+	assertStatus(t, response, http.StatusOK)
+
+	// TODO: Should be fine but may need to add a wait method to slow down resync op
```

Comment:
Is this prone to races, as described in the TODO?

Reply:
I think it may theoretically be but I'm not sure it'd be possible to re-run the sync fn on 1000 docs before a http call can be made. I think its safe to leave this as is.